citesphere/src/main/webapp/WEB-INF/views/auth/group/items.html

```diff
@@ -201,8 +204,137 @@ <h2>
             $("#syncProgress").removeClass("progress-bar-striped active");
             $("#syncText").text("Up to date");
         }
+        getCurrentItemsData();
     });
 }
+
+function getCurrentItemsData(){
+let pageNumber = $(".page-item.active").text();
+$.get("items/data?page="+pageNumber, function(data) {
+	if(!data['notModified']){
+		rewriteCitationsTable(data);
+		rewriteCollectionsList(data);
+		rewriteGroupInfo(data);
+		addEventListeners();
+		addCheckboxesEventListeners();
+		addMoveItemListeners();
+	}
+}); 
+}
+
+function rewriteCitationsTable(data){
+	let groupId = data['zoteroGroupId']
+	let collectionId = data['collectionId']
+	
+	let tablebody = $('<tbody>');
+	let headerRow = $('<tr>'+
+		'<th title="Move to collection">'+ 
+		'<div class="btn-group">' +
+		'<button id="btnGroupDrop1" type="button" class="btn btn-secondary dropdown-toggle icon-move-to" style="font-size:160%"  data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">' +
+		'<span class="caret"></span></button>'+
+		'<div class="dropdown-menu" aria-labelledby="btnGroupDrop1">' +
+		'<li id="selectAll" class="dropdown-item"><a>All</a></li>'+
+		'<li id="deSelectAll" class="dropdown-item"><a>None</a></li>' +
+		'</div>'+
+		'</div>' +
+		'</th>'+
+		'<th>'+"Type"+'</th>'+
+		'<th>'+"Author"+'</th>'+
+		'<th>'+"Title"+'</th>'+
+		'<th>'+"Date"+'</th>'+
+		'<th>'+"URL"+'</th>'+
+		'</tr>')
```

Comment:
the header row doesn't change, does it? So it wouldn't need to be added every time.